Paperboy/Example/Example.html

```diff
@@ -0,0 +1,13 @@
+---
+title: Title goes here          # {string}  required - Title of the message
+author: Your name 				# {string}  optional - Name of the author
+type: information				# {string}  required - Type of message (information, extension_updates, weekly_news, developers, important)
+version: 1						# {integer} optional - Version of the message, increment if changes have been made to the file
+date: 0                         # {integer} optional - Date as Unix timestamp
```

Comment:
I know. I tried fixing it in both Visual studio as well as Notepad++ but it's just not working